haproxy/state.go

```diff
@@ -79,6 +79,16 @@ func (h *HAProxy) watch(sd *lib.Shutdown) error {
 				waitAndRetry()
 				continue
 			}
+			// Don't check spoe filter index since it is incremented on dataplane api because retrieved using filters() routine
+			// But requires it to be set to 0 when initializing since filter are created one by one via filter() call
+			for index, current_state_frontend := range currentState.Frontends {
+				if current_state_frontend.FilterSpoe != nil {
+					fromHa.Frontends[index].FilterSpoe.Filter.Index  = current_state_frontend.FilterSpoe.Filter.Index
+				}
```

Comment:
Current code uses lowerCamelCase for identifiers, could you please use consistent case?

Reply:
Done